polylogue/providers/drive.py

```
@@ -0,0 +1,41 @@
+
+    name = "drive"
+    title = "Google Drive"
+
+    def __init__(self, ui: UI, client_factory: Callable[[UI], DriveClient] = DriveClient):
+        self._client = client_factory(ui)
+
+    @property
+    def ui(self) -> UI:
+        return self._client.ui
+
+    def __getattr__(self, item: str):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__getattr__`

(ANN204)